src/payment/strategies/mollie/mollie.ts

```diff
@@ -0,0 +1,41 @@
+export interface MollieClient {
+    createComponent(type: string, options?: object): MollieElement;
+    createToken(): Promise<MollieToken>;
+}
+
+export interface MollieToken {
+    token: string;
+    error: object;
+}
+
+export interface MollieConfigurationOptions {
+    locale: string;
+    testmode: boolean;
+}
+
+export interface MollieHostWindow extends Window {
+    Mollie?(
```

Comment:
`Mollie?` should this not be optional?

Reply:
if i remove the optional sign i get a warning 

_**Property 'Mollie' is missing in type 'Window & typeof globalThis' but required in type 'MollieHostWindow'.ts(2741)**_